cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminRunTime.feature

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Sysadmin
+Feature: Sysadmin - Validate system admin page Run time scenarios
+
+  @Sysadmin
+  Scenario:To verify user should be able to create Namespace successfully in System Admin
+    Given Open Datafusion Project to configure pipeline
+    Then Open "System Admin" menu
+    Then Click on the Configuration link on the System admin page
+    Then Click on Create New Namespace button
+    Then Enter the New Namespace Name with value: "namespaceName"
+    Then Enter the Namespace Description with value: "validNamespaceDescription"
+    Then Click on: "Finish" button in the properties
+    Then Verify the namespace created success message displayed on confirmation window
+    Then Verify the created namespace: "namespaceName" is displayed in Namespace tab
+
+  @Required
+  Scenario:To verify User should be able to add a secure key from Make HTTP calls successfully with PUT calls
+    Given Open Datafusion Project to configure pipeline
+    Then Open "System Admin" menu
+    Then Click on the Configuration link on the System admin page
+    Then Click on Make HTTP calls from the System admin configuration page
+    Then Select request dropdown property with option value: "httpPutMethod"
+    Then Enter input plugin property: "requestPath" with value: "secureKey"
+    Then Enter textarea plugin property: "requestBody" with value: "bodyValue"
+    Then Click on send button
+    Then Verify the status code for success response
+
+  @Required
```

Comment:
Can we change the name of the Required tag something like SysAdminRequired?

Reply:
Changed to SysAdminRequired.